en/contributing/build.md

```diff
@@ -82,37 +87,43 @@ To build the *DroneCore* C++ Library on Windows:
      cmake --build . --config Release
      ```
 
-1. (Optionally) "Install" DroneCore [as described below](#install-artifacts). This is required in order to build DroneCore applications, but not to run DroneCore test code.
+1. (Optionally) "Install" DroneCore [as described below](#install-artifacts). This is required in order to build [Dronecore C++ apps](../guide/toolchain.md), but not to run DroneCore test code.
 
 
 ## Install DroneCore {#install-artifacts}
 
-In order to *use* DroneCore your C++ applications need to be able locate the DroneCore libraries and header files (see [Building C++ Apps](../guide/toolchain.md)). You can either install these to standard system locations or locally within the DroneCore directory.
-
-> **Tip** The example code assumes that DroneCore is installed locally. The example **CMakeLists.txt** file can easily be modified to find the system-wide installation.
+*Installing* builds DroneCore **and** copies the libraries and header files into a "public" location so that they can be referenced by C++ applications (see [Building C++ Apps](../guide/toolchain.md)). DroneCore supports installation system-wide (recommended) or locally within the DroneCore tree.
 
+> **Warning** System-wide installation is not yet supported on Windows (see [#155](https://github.com/dronecore/DroneCore/issues/155)) so you will need to [install DroneCore locally](#dronecore_local_install).
+>
+> Windows gurus, we'd [love your help](../README.md#getting-help) to implement this).
 
-### System-wide Install
+### System-wide Install {#dronecore_system_wide_install}
 
-Installing DroneCore system-wide means that the files you need to build your apps are always in the same place, and your build definition files can be simpler because they do not need to consider relative locations.
+System-wide installation copies DroneCore to the standard system-wide locations for your platform (On Ubuntu Linux this is **/usr/local/**).
 
-> **Warning** System-wide install is not (yet) supported on Windows. If you're a Windows guru, we'd [love your help](../README.md#getting-help) to set this up.
+> **Warning** System-wide installation overwrites any previously installed version of DroneCore. 
 
-To install the files system-wide:
+To install DroneCore system-wide:
 
 ```sh
-make clean  #REQUIRED!
-sudo INSTALL_PREFIX=/usr/local make default install
+sudo make clean  #REQUIRED!
```

Comment:
Is clean still required. Is sudo required on both for this case.

Reply:
It is advised to do the make step without sudo and only the install step with sudo. This way `make clean` works without sudo.

Reply:
Ah, thanks for the clarification.